charts/velocity-operator/values.yaml

```diff
@@ -47,11 +47,12 @@ secrets:
   authToken: replace-me
 
 aws:
-  region: &aws-region us-east-1
+  region:
   creds:
     accessKey:
     secretKey:
 
+    # Note that anchor values are resolved ONCE and will not be updated on 'helm upgrade'
     k8sSecretName: &k8s-secret-name aws-creds
     k8sSecretKeyName: &k8s-secret-key-name credentials
     profile: &aws-profile default
```

Comment:
The point here was to allow the customer to override the default in case they want to store the secret on their own.

If YAML anchors don't allow that, please change this to be a value (yes, I understand this value will need to be repeated for each controller. It's a bummer, but since it's a custom configuration, we need to allow it)